pwintools.py

```diff
@@ -609,13 +615,14 @@ def get_timeout(self):
         return -1
 
     def set_timeout(self, timeout):
-        if timeout:
+        pass

```

Comment:
I believe that this change breaks the process timeout for r/w, what do you propose ?

Reply:
I suggest to let as it is and then merge in upstream to pwntools to reuse the native implementation of pwntool.